components/logins/src/engine.rs

```diff
@@ -201,92 +199,150 @@ mod test {
             ..Login::default()
         };
 
+        // Test to make sure that we can add items a and b to the engine
```

Comment:
All of these little comments could be replaced by a more general comment at the beginning of the test. e.g.,

```
// Adds two logins to the database and validates that they can be retrieved, then
// updates one of the logins and validates the retrieved value.
```